README.md

```diff
@@ -26,54 +38,79 @@ https://hub.docker.com/_/nats/
 You can find the instructions for the NATS Cli here:
 https://github.com/nats-io/natscli
 
+A `docker-compose` file is also available for standing up NATS Jetstream server.
+
 ### Create the HL7 stream and MESSAGES consumer
```

Comment:
Nit – not calling it `MESSAGES` anymore right?

Reply:
Fixed.